### ISSUE TITLE
Sober instantly aborts with no logs if ClientAppSettings.json is formatted incorrectly, returns error code 134

### ISSUE THREAD

**Issue by Macosaro** [vinegarhq/sober#97]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Here's an example of an incorrectly formatted ClientAppSettings.json file (commas are missing in some places):
```json
{
    "DFFlagDisableDPIScale": true,
    "DFIntTaskSchedulerTargetFps": 0
    "FFlagAdServiceEnabled": false
    "FFlagDebugDisableTelemetryEphemeralCounter": true,
    "FFlagDebugDisableTelemetryEphemeralStat": true,
    "FFlagDebugDisableTelemetryEventIngest": true,
    "FFlagDebugDisableTelemetryPoint": true,
    "FFlagDebugDisableTelemetryV2Counter": true
    "FFlagDebugDisableTelemetryV2Event": true,
    "FFlagDebugDisableTelemetryV2Stat": true,
    "FFlagFutureIsBrightPhase3Vulkan": true,
    "FFlagGameBasicSettingsFramerateCap5": true,
    "FFlagRealMobileOutline": false,
    "FFlagUserHandleChatHotKeyWithContextActionService": true
}
```
Trying to run Sober with such settings makes it abort instantly.

**Comment by km9l** [comment]
this is a bug how?

**Comment by tunis4** [comment]
it should at least print an error message instead of just segfaulting

**Comment by Macosaro** [comment]
Was fixed in `0.0.0-7a34212`.
